docs/upgrading/upgrading-to-4.3.md

```diff
@@ -1,5 +1,27 @@
 # Rundeck 4.3 Upgrade Notes
 
 :::warning
-Any keys or passwords created using encryption in 4.2.0/4.2.1 will be lost when upgrading, it is recommended to back up your data before upgrading from those versions. Data created in earlier versions (before v4.2.0) is will transfer properly.
-:::
\ No newline at end of file
+When upgrading from 4.1 to 4.2, some users reported keys and passwords created using encryption were lost. Therefore, we strongly recommend backing up your data before upgrading from 4.2 to 4.3. Data created prior to version 4.2 should not be affected when upgrading to 4.3.
+:::
+
+## Private keys and password encryption issue
+
+The release 4.3.0 fixes an issue that results in invalid credentials after upgrading from version 3.4.x to version 4.2.0.
+
+This issue is related to the key encryption method and causes keys already stored using previous versions to become invalid.
+
+Specifically, private keys used in SCM becoming invalid after the upgrade. This results in disabling all projects at that use SCM with SSH private keys.
+
+## Upgrading from 4.2.0
+
+This issue was detected after upgrading to v4.2.x from any previous version.
+
+- If you didn't change the stored private keys after the upgrading then you don't need to do anything in v4.3.0.
```

Comment:
Is it only "change" an existing key?   Or is it "If you didn't add/change any keys stored in the Key Storage..."?